draft-ietf-mls-architecture.md

```diff
@@ -607,9 +606,9 @@ including total DoS attacks (where it simply refuses to forward any
 messages) and partial DoS attacks (where it refuses to forward
 messages to and from specific Clients). As noted in
 {{delivery-guarantees}}, these attacks are only partially detectable
-by clients. Ultimately, failure of the DS to provide reasonable
-service must be dealt with as a customer service matter, not via
-technology.
+by clients without an out-of-band channel. Ultimately, failure of
```

Comment:
What do you mean here ?

Reply:
It feels like too strong an assertion to claim that there can be no technical solution to detecting a targeted DoS by the DS. Something out of band might work; such as a transparency protocol.